Source/engine/points_in_rectangle_range.hpp

```diff
@@ -228,22 +236,35 @@ class PointsInRectangleRange {
 	}
 
 protected:
-	Rectangle region;
+	RectangleOf<CoordT> region;
 };
 
-class PointsInRectangleRangeColMajor {
+template <typename CoordT>
+PointsInRectangleRange<CoordT> PointsInRectangle(RectangleOf<CoordT> region)

```

Comment:
pretty sure this can be constexpr/inline

Reply:
Added `constexpr`. Template functions are always inline.